content/php/concepts/math-functions/terms/hypot/hypot.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'hypot()'
+Description: 'Returns the length of the hypotenuse of a right-angle triangle.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Geometry'
+  - 'Math'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`hypot()`** function returns the length of the hypotenuse of a right-angle triangle. Given a right-angle triangle with sides `$length1` and `$length2`, it returns the square root of (`$length1`<sup>2</sup> + `$length2`<sup>2</sup>).
+
+The function accepts two `float` inputs and returns a value of type `float`.
+
+## Syntax
+
+```pseudo
+$hypotenuse = hypot(length1, length2);
+```
+
+Where `length1` and `length2` are the two lengths of a right-angle triangle.
+
+## Example
+
+This example calculates the hypotenuse of a right-angle triangle with sides of length `6` and `8`.
+
+```php
+<?php
+  $hypotenuse = hypot(6, 8);
+  echo $hypotenuse;
+?>
+```
+
+This example results in the following output:
+
+```shell
+10
+```
+
+## Codebyte Example
+
+This example returns the hypotenuse for two right-angle triangles - one with sides whose lengths are expressed in whole numbers, and one with sides whose lengths contain a fractional component:
```

Comment:
This example is largely redundant. Codebytes are optional. Please revise this example to demonstrate some alternative functionality or delete it.

Reply:
@caupolicandiaz I have removed the comparison of two 90-degree triangles, and just left one example that Docs viewers can run to ensure the function works as expected (a 90-degree triangle with sides of length `12` and `16`).  Personally, I like to run a Codebyte snippet to ensure that I have understood the concept correctly.  I am happy to totally remove this Codebyte if you would prefer.